src/main/java/org/shanerx/tradeshop/shop/ShopSign.java

```diff
@@ -53,7 +54,8 @@ enum Signs {
     CRIMSON_WALL_SIGN("1.16.0", ""),
     WARPED_WALL_SIGN("1.16.0", "");
 
-    private List<Integer> minVer = Arrays.asList(new Integer[3]), maxVer = Arrays.asList(new Integer[3]);
+    private final List<Integer> minVer = Arrays.asList(new Integer[3]);
```

Comment:
Final thing, seems the version i'm on checks and auto formats this in. I would assume if it does so then they could be final so none of these should cause problems.